src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -64,21 +64,28 @@
 VERSION_FILE = 'versions.yaml'
 COMPUTE_TOOLS_DIR = '../../compute_tools'  # relative to task_generator.py
 
-# TODO GTDB update readme to specify to get correct version of data, not just latest
-# TODO REPRODUCIBILITY need to version the databases and use the correct version with the
-#                      correct tool version
-# directory containing the unarchived GTDB-Tk reference data
-# download data following the instructions provided on
-# https://ecogenomics.github.io/GTDBTk/installing/index.html#gtdb-tk-reference-data
-GTDBTK_DATA_PATH = '/global/cfs/cdirs/kbase/collections/libraries/gtdb_tk/release207_v2'
+# volume name for the Docker containers
+TOOL_IMG_VOLUME_NAME = {'checkm2': '/CheckM2_database',
```

Comment:
Is this a mapping from the tool name to the internal container ref data mount directory?

Reply:
yes. Added comments. Or using something like `reference_data` for both tool images instead of using a different name?

Reply:
Yeah, that seems good. Then you can get rid of the map and trigger the mounts based on whether there's a ref data version in the version file

Reply:
👍

Reply:
:+1: